core/templates/.spacemacs.template

```diff
@@ -80,6 +80,8 @@ values."
    ;; Possible values are: `recents' `bookmarks' `projects'.
    ;; (default '(recents projects))
    dotspacemacs-startup-lists '(recents projects)
+   ;; If non nil then checking for updates is enabled
+   dotspacemacs-version-check-enable t
```

Comment:
It's maybe not the best place to put it, as it's between two related settings. I would have placed it after `dotspacemacs-elpa-https`